Policies/Kubernetes/container-no-privilege-escalation/examples/example-good.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Can you also please add another YAML file where no security context specified in YAML.

Reply:
As you are doing thorough testing of PSP policies, it will be good to check-in different combinations of test files. In future if someone touches these policies, they can use the test files.

Reply:
REGO unit tests can cover exhausted list of tests, YAML files can cover basic tests.

Reply:
What do you mean by thorough testing? I'm testing a violation, a pass, and the namespace exclusion. I figured the unit tests would cover the rest.

For this policy, when no security context is specified the policy will pass as the default for that setting is false

Reply:
[This line](https://github.com/Azure/Community-Policy/blob/338e07b9a23e291c9dee32f65de1de776010e1d4/Policies/Kubernetes/container-no-privilege-escalation/rego/src.rego#L13) specifies the given container is considered in violation of the policy if allowPrivilegeEscalation is not explicitly set to false.

Reply:
@sozercan Should we show violation if allowPrivilegeEscalation  property not set on a container? I assume this property is not used in general by most of the conatiners which means we flag all of them.